optaplanner-docs/src/modules/ROOT/pages/design-patterns/design-patterns.adoc

```diff
@@ -70,10 +68,11 @@ In the following diagram, each row is a different example and shows the relation
 
 image::design-patterns/entityVariableAndValueExamples.png[align="center"]
 
-[NOTE]
-====
-Vehicle routing is different because it uses a xref:planner-configuration/planner-configuration.adoc#chainedPlanningVariable[chained planning variable].
-====
+NOTE: Course scheduling is different because it uses two xref:planner-configuration/planner-configuration.adoc#planningVariable[planning variables].
+
+NOTE: Vehicle routing is different because it uses a xref:planner-configuration/planner-configuration.adoc#planningListVariable[planning list variable].
+
+NOTE: Traveling salesmen is different because it uses a xref:planner-configuration/planner-configuration.adoc#chainedPlanningVariable[chained planning variable].
```

Comment:
```suggestion
[NOTE]
====
Course scheduling is different because it uses two xref:planner-configuration/planner-configuration.adoc#planningVariable[planning variables].
Vehicle routing is different because it uses a xref:planner-configuration/planner-configuration.adoc#planningListVariable[planning list variable].
Traveling salesman is different because it uses a xref:planner-configuration/planner-configuration.adoc#chainedPlanningVariable[chained planning variable].
====
```